src/aat/resources/features/F-103/F-103_Test_Case_Access_Base_Steve_C3.td.json

```diff
@@ -0,0 +1,30 @@
+{
+	"_guid_": "F-103_Test_Case_Access_Base_Steve_C3",
+	"method": "PUT",
+	"uri": "/cases/{cid}/users/{uid}",
+	"users": {
+    "invokingUser": {
+      "username": "befta.caseworker.caa.1@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    },
+    "userRichard": {
+      "username": "befta.caseworker.2.solicitor.2@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    },
+    "userDil": {
+      "username": "befta.caseworker.2.solicitor.3@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    },
+    "userSteve": {
+      "username": "befta.caseworker.2.solicitor.4@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    }
```

Comment:
in general, most of these repeated lines could be avoided by inheriting them from appropriate base objects. pls could you revise all test data files from this perspective, to maximise re-use and minimise repeated test data lines?

Reply:
done!!